flag_test.go

```diff
@@ -356,3 +356,33 @@ func TestNoInterspersed(t *testing.T) {
 		t.Fatal("expected interspersed options/non-options to fail")
 	}
 }
+
+func TestTermination(t *testing.T) {
+	f := NewFlagSet("termination", ContinueOnError)
+	if f.Parsed() {
+		t.Error("f.Parse() = true before Parse")
+	}
+	arg1 := "ls"
+	arg2 := "/home"
```

Comment:
Problem is that these would normally not really be confused for flags... Why not use something like `arg2 := "-l"` instead? And possibly register a conflicting `-l` flag of your own and confirm that it's not being triggered. Ideally, I'd want a test that would *fail* had the commit to the code not be included but this one doesn't seem to...